index.bs

```diff
@@ -10775,21 +10775,15 @@ in the algorithm of <a href="#rendering-a-graph">rendering a graph</a>.
 <div class="note">
 	In practice, the {{AudioContext}} <a>rendering thread</a> is
 	often running off a system-level audio callback, that executes in
-	an isochronous fashion. This callback passes in a buffer that has
-	to be filled with the audio that will be output. The size of the
-	buffer may be larger than a rendering quantum. In this case,
-	multiple invocations of the rendering algorithm will be called in a
-	rapid succession, in the same callback, before returning. After
-	some time, the underlying audio system will call the callback
-	again, and the algorithm will be executed again. This is an
-	implementation detail that should not be observable, apart from the
-	latency implications.
-
-	{{OfflineAudioContext}} will execute the algorithm continuously,
-	until {{OfflineAudioContext/length}} frames have been rendered.
+	an isochronous fashion. 
+
+	An {{OfflineAudioContext}} normally doesn't have this
```

Comment:
What does it mean by "normally" here?

Reply:
Maybe we can say an oac is not required to a system-level audio callback?

Reply:
PTAL at the updated sentence.